CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Precise UI Changelog
 
+## 2.1.6
+
+- Updated `Form` component to have onValidate callback
```

Comment:
I think we did not release 2.1.5 yet -> you can just put it there.

Reply:
Thank you, I removed increasing of version